features/contract-state/GeneralInfo.tsx

```diff
@@ -64,8 +64,8 @@ const GeneralInfo = () => {
     const expiryDate = new Date(
       expiry.toNumber() * 1000
     ).toLocaleString("en-GB", { timeZone: "UTC" });
-    const prettyLatestPrice = Number(latestPrice).toFixed(4);
-    const pricedGcr = (gcr / latestPrice).toFixed(4);
+    const prettyLatestPrice = Number(latestPrice).toFixed(8);
```

Comment:
This is to account for an identifier like USDBTC where the price is 0.000085... which requires > 4 decimals